# Summary Statistics: Generates descriptive statistics that summarize the central tendency, dispersion and shape of a dataset’s distribution, excluding NaN values.
## <html> <font color='green'>Summary statistics usually comprises of the 5 Number Summary viz..</font></html>
## 1. Minimum
## 2. 1st. Quartile (25 percentile - i.e. 25% of the data falls below this value)
## 3. Median (2nd Quartile) (50 percentile - i.e. 50% of the data falls below this value)
## 4. 3rd. Quartile (75 percentile - i.e. 75% of the data falls below this value)
## 5. Maximum 

In [2]:
import pandas as pd
df = pd.read_csv("Churn_Modelling.csv")
pd.DataFrame.describe(df.Age)


count    10000.000000
mean        38.921800
std         10.487806
min         18.000000
25%         32.000000
50%         37.000000
75%         44.000000
max         92.000000
Name: Age, dtype: float64

# <html><font color='Blue'>Inference:</font></html>
## Minimum age of the person: <html><font color='red'>18 years</font></html>
## Maximum age of the person: <html><font color='red'>92 years</font></html>
## 25% of the people have an age < <html><font color='red'>32 years </font></html>
## 50% of the people have an age < <html><font color='red'>37 years </font></html>
## 75% of the people have an age < <html><font color='red'>44 years </font></html>

# Additional summary statistics:
## 1. Range : Maximum - Minimum = <html><font color='red'> 92 - 18 = 74 </font></html>
## 2. IQR: Inter Quartile Range = <html><font color='red'> Q3 -Q1 = 44 - 32 = 12 </font></html>
## 3. SD: <html><font color='red'> 10.487806 (Age) </font></html>i.e. on an average the age of the person will be 10.487806 above or below the mean average age. The SD measures how FAR the value is from the mean. According to the Emperical Rule or the 68-95-99.7 Rule:
### 68% of the observations lie within +/-1 SD of the mean. 
### 95% of the observations lie within +/- 2 SD of the mean. 
### 99.7% of the observations lie within +/- 3 SD of the mean. 

## 4. Mean: <html><font color='red'> 38.921800 Age </font></html> i.e. roughly 2 units more than the median or the 50th percentile or Q2.